modes/magit/README.org

```diff
@@ -0,0 +1,215 @@
+   | copy section info      | =C-w=   |                        | =ys=                  |               |          |
+   | copy buffer info       | =M-w=   |                        | =yb=                  |               |          |
+
+** New Commands
+
+   | Command                     | EM w/o yank opt          | EM w/ yank opt (default) | w/ horiz move |
+   |-----------------------------+--------------------------+--------------------------+---------------|
+   | evil-goto-line              | =G=                      |                          |               |
+   | evil-next-visual-line       | =j=                      |                          |               |
+   | evil-previous-visual-line   | =k=                      |                          |               |
+   | evil-backward-char          | under =M-x=              |                          | =h=           |
+   | evil-forward-char           | under =M-x=              |                          | =l=           |
+   | evil-search-next            | =n=                      |                          |               |
+   | evil-search-previous        | =N=                      |                          |               |
+   | set-mark-command            | =v= or =V=               | =C-SPC=                  |               |
+   | evil-visual-line            | under =M-x=              | =v= or =V=               |               |
+   | evil-ex                     | =:=                      |                          |               |
+   | evil-search-forward         | =/=                      |                          |               |
+   | evil-scroll-page-up         | =C-b=                    |                          |               |
+   | evil-scroll-down            | =C-d=                    |                          |               |
+   | evil-scroll-page-down       | =C-f=                    |                          |               |
+   | evil-scroll-up              | =C-u= (if =C-u= scrolls) |                          |               |
+   | evil-emacs-state            | =C-z=                    |                          |               |
+   | evil-yank-line              | under =M-x=              | =yy=                     |               |
+   | evil-window-map             | under =M-x=              | =C-w=                    |               |
+   | evil-collection-magit-toggle-text-mode | =C-t/\=                  |                          |               |
+
+
+Any other bindings are meant to be consistent with these.
+
+Use =evil-collection-magit-revert= to revert changes made by evil-collection-magit to the default
+evil+magit behavior.
+
+** To add other common evil commands
+
+Some may want =?= to search backward instead of launching the popup which is
+also bound to =h=. To get this behavior, add the following line after =(require
+'evil-collection-magit)= in your configuration.
```

Comment:
What if we just require 'evil-collection?

Reply:
Changed it to use evil-collection-init.